netconfig.md

```diff
@@ -12,7 +12,7 @@ You need to manually permit the Spacemesh App to access the internet by opening
 1. Navigate to Control Panel, System and Security and Windows Firewall.
```

Comment:
1. In my computer it says "Windows Defender Firewall".
2. It depends how the Control Panel looks like. I don't have Category View, so I don't have System and Security.

Reply:
I updated the instructions in this branch - pull.